### PR TITLE
New parameter --defaultcrop to define genmeta crop factor

### PR DESCRIPTION
**Pull request by ghost**
The `.stitch` file can always be edited, but I wanted to have a better crop default value without recalculating the the width/height of the crop box manually. I think 10% is too much, so I introduce a new parameter allowing to specify the crop factor with an additional parameter `--defaultcrop` that can be combined with `--genmeta`.

### CODE REVIEW

File: pdfstitch [hunk 1]
```diff
@@ -41,14 +42,17 @@ GetOptions
 (
     'h|help' => \$help,
     'g|genmeta' => \$genmeta,
+    'dc|defaultcrop=s' => \$defaultcrop,
```
Comment (sur5r) [inline pdfstitch]: "dc" will not work for short options. This has to be a single character. I'd suggest 'd'.

**Comment by sur5r** [comment]
Maybe ``--defaultcrop`` should be mentioned in ``$usage`` as well.

File: pdfstitch (hunk 1 above)
Comment (ghost) [reply pdfstitch]: thx, done.

**Comment by ghost** [comment]
I've added the parameter to `$usage` as well.

**Pull request merged**